package-lock.json

```diff
@@ -1,7959 +1,8036 @@
-{
-  "name": "setup-go",
-  "version": "1.0.0",
-  "lockfileVersion": 2,
-  "requires": true,
-  "packages": {
-    "": {
-      "name": "setup-go",
-      "version": "1.0.0",
-      "license": "MIT",
-      "dependencies": {
-        "@actions/core": "^1.6.0",
-        "@actions/http-client": "^1.0.6",
-        "@actions/io": "^1.0.2",
-        "@actions/tool-cache": "^1.5.5",
-        "semver": "^6.1.1"
-      },
-      "devDependencies": {
-        "@types/jest": "^27.0.2",
```

Comment:
@nikeee looks like the line endings on this file changed to `CRLF`.  Can you change it back to `LF`?

@vsafonkin we probably should add `* text=auto` to https://github.com/actions/setup-go/blob/main/.gitattributes